sbin/prepareWorkspace.sh

```diff
@@ -290,35 +290,46 @@ updateOpenj9Sources() {
     OPENJCEPLUS_FLAGS=""
     GSKIT_FLAGS=""
     GSKIT_CREDENTIALS=""
-    if [ "$JAVA_FEATURE_VERSION" -eq 17 ]
-    then
-      if [ "${BUILD_CONFIG[BUNDLE_OPENJCEPLUS]}" == "true" ]; then
-        # Set the flags to get the OpenJCEPlus source code
-        OPENJCEPLUS_BRANCH="semeru-main"
-        # Set the flags to get the appropriate GSKit binaries
-        GSKIT_FOLDER="https://na.artifactory.swg-devops.com/artifactory/sec-gskit-javasec-generic-local/gskit8/20230802_8.9.5"
-        if [ "$TARGET_OS" = "linux"  ]; then
-          if [ "$ARCHITECTURE" = "x64"  ]; then
-            OPENJCEPLUS_FLAGS="-openjceplus-repo=https://github.com/ibmruntimes/OpenJCEPlus.git -openjceplus-branch=${OPENJCEPLUS_BRANCH}"
-            GSKIT_FLAGS="-gskit-bin=${GSKIT_FOLDER}/linux64_x86/jgsk_crypto.tar -gskit-sdk-bin=${GSKIT_FOLDER}/linux64_x86/jgsk_crypto_sdk.tar"
-            GSKIT_CREDENTIALS="-gskit-credential=$GSKIT_USERNAME:$GSKIT_PASSWORD"
-          elif [ "$ARCHITECTURE" = "ppc64le"  ]; then
-            OPENJCEPLUS_FLAGS="-openjceplus-repo=https://github.com/ibmruntimes/OpenJCEPlus.git -openjceplus-branch=${OPENJCEPLUS_BRANCH}"
-            GSKIT_FLAGS="-gskit-bin=${GSKIT_FOLDER}/linux64_ppcle/jgsk_crypto.tar -gskit-sdk-bin=${GSKIT_FOLDER}/linux64_ppcle/jgsk_crypto_sdk.tar"
-            GSKIT_CREDENTIALS="-gskit-credential=$GSKIT_USERNAME:$GSKIT_PASSWORD"
-          fi
+    
+    if [ "${BUILD_CONFIG[BUNDLE_OPENJCEPLUS]}" == "true" ]; then
+      SUPPORTED_PLATFORM=false
+      if [ "$TARGET_OS" = "linux"  ]; then
+        if [ "$ARCHITECTURE" = "x64" ] || [ "$ARCHITECTURE" = "ppc64le" ] || [ "$ARCHITECTURE" = "s390x"  ]; then
+          SUPPORTED_PLATFORM=true
         fi
-        if [ "$TARGET_OS" = "aix"  ]; then
-          OPENJCEPLUS_FLAGS="-openjceplus-repo=https://github.com/ibmruntimes/OpenJCEPlus.git -openjceplus-branch=${OPENJCEPLUS_BRANCH}"
-          GSKIT_FLAGS="-gskit-bin=${GSKIT_FOLDER}/aix64_ppc/jgsk_crypto.tar -gskit-sdk-bin=${GSKIT_FOLDER}/aix64_ppc/jgsk_crypto_sdk.tar"
+      elif [ "$TARGET_OS" = "aix" ] || [ "$TARGET_OS" = "windows" ]; then
+          SUPPORTED_PLATFORM=true
+      fi
+
+      if [ "$SUPPORTED_PLATFORM" == "true" ]; then
+        # Set the flags to get the OpenJCEPlus source code
+        OPENJCEPLUS_BRANCH="semeru-java$JAVA_FEATURE_VERSION"
+        OPENJCEPLUS_FLAGS="-openjceplus-repo=https://github.com/ibmruntimes/OpenJCEPlus.git -openjceplus-branch=${OPENJCEPLUS_BRANCH}"
+      fi
+      
+      
+      # Set the flags to get the appropriate GSKit binaries
```

Comment:
Is it cleaner to write these in 2 sections or would it be more efficient to have the gskit variables setup in the previous set of `if`'s? I still like having the SUPPORTED_PLATFORM variable so that the other flags only need to be set once.